### PR TITLE
rental marketplace pt 1

### PR DESCRIPTION
**Pull request by emilyJLin95**

### CODE REVIEW

File: src/guides/nft-rental-marketplace/index.md [hunk 1]
```diff
@@ -0,0 +1,1547 @@
...
+```shell
+truffle dashboard
+truffle migrate --network dashboard
+```
+
+This should bring up Truffle dashboard on localhost:24012 and prompt you to sign the deployment. After that's been deployed, now let's verify it!
+
+### Verify RentablePets
+
+Follow these steps here: https://github.com/rkalis/truffle-plugin-verify
```
Comment (kevinbluer) [inline src/guides/nft-rental-marketplace/index.md:1011]: Might be worth making this a link

File: src/guides/nft-rental-marketplace/index.md [hunk 2]
```diff
@@ -0,0 +1,1547 @@
...
+
+We recommend verifying with dashboard:
+
+`truffle run verify RentablePets --network dashboard`
+
+You might run into issues with the OpenZepplin contracts being in the `node_modules` folder. If so, go ahead and copy the necessary ones from `node_modules` into the `contracts` folder and change the import paths in `RentablePets.sol` and `ERC4907.sol`. Then you'll need to redeploy and follow the verify steps again.
+
+If you would like, just clone the `verify-contract` branch of the unleashed repo instead: https://github.com/trufflesuite/unleashed_rentable_nft/tree/verify-contract
```
Comment (kevinbluer) [inline src/guides/nft-rental-marketplace/index.md:1019]: Same with this link

File: src/guides/nft-rental-marketplace/index.md [hunk 3]
```diff
@@ -0,0 +1,1547 @@
...
+    } catch(err) {
+        console.log(err);
+    }
+    cb();
+}
+  
+module.exports = main;
+```
+
+You might not have enough eth in the owner wallet to cover the refund. If so, jump to the bonus section to write a script to fund wallets with ETH on your ganache instance!
```
Comment (kevinbluer) [inline src/guides/nft-rental-marketplace/index.md:1470]: There's a bit of inconsistency in both this paragraph (and throughout other areas of the guide) when referencing ETH. Personally I'm a fan of all caps 🙃

**Review by kevinbluer — CHANGES_REQUESTED** [review]
A bunch of minor things. Overall content, etc looks great though!

File: src/guides/nft-rental-marketplace/index.md [hunk 4]
```diff
@@ -0,0 +1,1547 @@
...
+
+1. Starting a project using the Truffle React box to prepare for building a frontend in part 2
+2. Writing a rental marketplace contract that will interact with rentable NFTs
+3. Deploying contracts that are dependent on each other
+4. Writing tests
+5. Verifying the `RentablePets` contract using [truffle-plugin-verify](https://github.com/rkalis/truffle-plugin-verify) on Goerli Etherscan
+6. Using Ganache forking and unlocking your MetaMask account locally to test the marketplace contract with already deployed contracts
+7. Writing Truffle scripts for fast testing
+
+<iframe width="560" height="315" src="https://youtu.be/LJGCkYC2ZCE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
[the remark this answers is not in the document]
Comment (emilyJLin95) [reply src/guides/nft-rental-marketplace/index.md:26]: Hmm yeah I just assumed it was cause it was in preview mode, but i'll try relinking it

[the remark this answers is not in the document]
Comment (kevinbluer) [reply src/guides/nft-rental-marketplace/index.md]: Awesome, looks good now 👍

**Review by kevinbluer — APPROVED** [review]
Awesome, LGTM now!

**Pull request merged**